support-frontend/assets/pages/[countryGroupId]/student/StudentLandingPageUTSContainer.tsx

```diff
@@ -9,25 +9,18 @@ import { ratePlanToBillingPeriod } from 'helpers/productPrice/billingPeriods';
 import { allProductPrices } from 'helpers/productPrice/productPrices';
 import { getPromotion } from 'helpers/productPrice/promotions';
 import { type GeoId } from 'pages/geoIdConfig';
+import { StudentLandingPageUTS } from './components/StudentLandingPageUTS';
 import { getStudentDiscount } from './helpers/discountDetails';
-import { StudentLandingPage } from './StudentLandingPage';
 
-export function StudentLandingPageContainer({
-	geoId,
-	productKey,
-	ratePlanKey,
+export function StudentLandingPageUTSContainer({
 	landingPageVariant,
 }: {
-	geoId: GeoId;
-	productKey: ActiveProductKey;
-	ratePlanKey: ActiveRatePlanKey;
 	landingPageVariant: LandingPageVariant;
 }) {
-	/**
-	 * Non-AU Students have ratePlanKey as OneYearStudent
-	 * AU Students have ratePlanKey as Monthly, productKey as SupporterPlus
-	 * and optional promoCode
-	 */
+	const geoId: GeoId = 'au';
+	const productKey: ActiveProductKey = 'SupporterPlus';
+	const ratePlanKey: ActiveRatePlanKey = 'Monthly';
+
 	const countryId: IsoCountry = Country.detect();
 	const promotionSupporterPlus = getPromotion(
```

Comment:
Wonder whether this should read `promotionUTSStudent` or `promotionSuppoterPlusUTSStudent`? Maybe a mouthfull!